src/shapes/polyline.class.ts

```diff
@@ -144,8 +144,10 @@ import { degreesToRadians } from '../util/misc/radiansDegreesConversion';
           top: bbox.top - legacyCorrection,
           pathOffset: new Point(pathOffsetX, pathOffsetY),
           strokeOffset: new Point(bboxNoStroke.left, bboxNoStroke.top).subtract(
-            bbox.left,
-            bbox.top
+            new Point(bbox.left, bbox.top)
```

Comment:
caught error

Reply:
we should add a unit test for this

Reply:
added here https://github.com/fabricjs/fabric.js/pull/8374/commits/b895215350215d0904a0febf9c248bfea8af0833, but i don't know if it's what you have in mind